tests/operator_tests.py

```diff
@@ -745,13 +750,14 @@ def test_str_repr(self):
         # Test __repr__()
         self.assertEqual("array([1 2 3])", ak.array([1, 2, 3]).__repr__())
         self.assertEqual("array([1 2 3 ... 17 18 19])", ak.arange(1, 20).__repr__())
-        self.assertEqual(
-            "array([1.1000000000000001 2.2999999999999998 5])", ak.array([1.1, 2.3, 5]).__repr__()
+        self.assertTrue(
+            "array([1.1000000000000001 2.2999999999999998 5])" == ak.array([1.1, 2.3, 5]).__repr__() or
+            "array([1.1 2.3 5])" == ak.array([1.1, 2.3, 5]).__repr__()
```

Comment:
same thing here as in the `__str__` case. Would be nice to update for readability but not pressing